.drone.star

```diff
@@ -2085,3 +2102,15 @@ def lintTest():
             "make test-lint",
         ],
     }]
+
+def waitForServer(federatedServerNeeded):
```

Comment:
In the activity app, waitForServer is before fixPermissions.
We need to copy exactly the code from the activity app (or update the activity app again). Otherwise the code becomes different in different apps, and then it gets harder to update it next time.

Reply:
@phil-davis can we update the placement in `activity` app because for every other app PR, this piece of code is pushed at the end of the file.

Reply:
yes, you can do that. But also please look at the other differences (see https://github.com/owncloud/activity/pull/1041 )
There are calls missing to `waitForServer` and `waitForEmailService`, places where the docker image name constants have not been used, etc.

Reply:
fixed this, can you review?